rep-I0001.rst

```diff
@@ -156,27 +158,40 @@ The motion interface can be expressed as four variations:
  
 Node Configuration
 ---------
-In order to support the various methods of control, the motion node must be somewhat dynamic/statically reconfigurable[see current parameters].  The node must be able to support subscriptions to multiple topics (all of the same type) as well as conversion from ROS group organizations to controller organization.  This mapping would look similar to the MoveIt controller manager[#ctrl_mgr]_.  
-The yaml file will contain a list of structures that defines the joint trajectory topics as well as the mapping to the controller.::
+In order to support the various methods of control, the motion node must be somewhat dynamic/statically reconfigurable[see current parameters].  The node must be able to support subscriptions to multiple topics (all of the same type) as well as conversion from ROS group organizations to controller organization.  This mapping would look similar to the MoveIt controller manager [#ctrl_mgr]_.  
+
+As in previous interfaces, the node will be configured using a ROS parameter specifying ROS-to-controller mapping.  This parameter will contain a list of structures that define both the topic namespaces as well as the mapping to the controller.::
 
-        topic_list:
-         - name: <topic name>
+        controller_joint_map:
+         - group: <controller group#>
            ns: <topic namespace>
-           group: <controller group#>
            joints:
             - <joint_1>
             - <joint_2>
             - <joint_N>
-         - name: <topic name>
+         - group: <controller group #>
            ns: ...
 
+This structure allows users to configure the robot interface to receive motion commands in several different ways:
+
+#. Single Topic
+    all groups use same namespace (e.g. `ns: ""`)
+#. Separate Topics
+    use group-specific namespaces
+#. Mixed Mode
+    joint map may contain multiple entries for the same group, in different namespaces.  This enables separate multi-arm topics for: left, right, both.
```

Comment:
I'd like to avoid referring to *left* and *right* as groups, as that is only relevant when dealing with dual-arm manipulators, while multi-group motion is used in many more setups. Perhaps just adding an *for example* to the sentence.